examples/with-federated-passkeys/src/pages/api/createSubOrg.ts

```diff
@@ -81,20 +76,25 @@ export default async function createUser(
     });
 
     const subOrgId = refineNonNull(
-      completedActivity.result.createSubOrganizationResultV3?.subOrganizationId
+      completedActivity.result.createSubOrganizationResultV4?.subOrganizationId
```

Comment:
commenting here but this is really for line 70 -- a tiny comment for the path could be helpful for those unfamiliar ("hey, this is the first/initial wallet -- hardcode to account 0")

Reply:
Good call! I'll make it a constant as well, for clarity.